GOVERNANCE.md

```diff
@@ -0,0 +1,50 @@
+OpenWDL Governance and Process
+==============================
+
+Table of Contents
+-----------------
+[Table of Contents]: #table-of-contents
+  - [Governance](#governance)
+  - [RFC Process](#rfc-process)
+  
+Governance
+----------
+
+WDL has a community-driven development process where most decisions are made through consensus, under the stewardship of a 'core team'. The core team is responsible for steering the design and development process, overseeing the introduction of new features, setting priorities and release schedule, and ultimately making decisions for the rare instances where there is no community consensus . 
+
+Current core team members are:
+
+| Name               | Organization            | github                  |
+|:-------------------|:-------------|:------------------
+| Brad Chapman       | Harvard School of Public Health | [chapmanb](https://github.com/chapmanb) |
+| Jeff Gentry        | Broad Institute                 | [geoffjentry](https://github.com/geoffjentry) |
+| Mike Lin           | DNANexus                        | [mlin](https://github.com/mlin) |
+| Patrick Magee      | DNAStack                        | [patmagee](https://github.com/patmagee) |
+| Brian O'Connor     | University of California, Santa Cruz | [briandoconnor](https://github.com/briandoconnor) |
+| Abirami Prabhakaran| Intel | [aprabhak2](https://github.com/aprabhak2) |
+| Geraldine Van der Auwera | Broad Institute | [vdauwera](https://github.com/vdauwera) |
+
+
+At the core group's discretion a new member may be added by a majority vote. This addition will be done to recognize **significant** contributions to the community. *Contributions* include such things as:
+
+ - Active participation in discussions, mailing list, forums
+ - Community building efforts
+ - Helping with documentation, standards documents, etc
+ - Building supporting software and tooling
+
+Generally members are only removed at their own request are due to very long term inactivity. In extreme circumstances the core group can vote to remove a member.
+
+RFC Process
```

Comment:
What does RFC stand for? You should write it out in full once, with (RFC) afterward. For example, Request for Contribution (RFC) process.